advanced/Templates/pihole-FTL-prestart.sh

```diff
@@ -10,22 +10,14 @@ utilsfile="${PI_HOLE_SCRIPT_DIR}/utils.sh"
 FTL_PID_FILE="$(getFTLConfigValue files.pid)"
 
 # Ensure that permissions are set so that pihole-FTL can edit all necessary files
-# shellcheck disable=SC2174
-mkdir -pm 0640 /var/log/pihole
+mkdir -p /var/log/pihole
 chown -R pihole:pihole /etc/pihole /var/log/pihole
-chmod -R 0640 /var/log/pihole
-chmod -R 0660 /etc/pihole
+find /etc/pihole/ /var/log/pihole/ -type d -exec chmod 0755 {} +
+find /etc/pihole/ /var/log/pihole/ -type f ! \( -name '*.pem' -o -name '*.crt' \) -exec chmod 0640 {} +
+find /etc/pihole/ /var/log/pihole/ -type f -name '*.pem' -o -name '*.crt' -exec chmod 0600 {} +
 
-# Logrotate config file need to be owned by root and must not be writable by group and others
+# Logrotate config file need to be owned by root
 chown root:root /etc/pihole/logrotate
-chmod 0644 /etc/pihole/logrotate
-
-# allow all users to enter the directories
-chmod 0755 /etc/pihole /var/log/pihole
-
-# allow pihole to access subdirs in /etc/pihole (sets execution bit on dirs)
-# credits https://stackoverflow.com/a/11512211
-find /etc/pihole/ -type d -exec chmod 0755 {} \;
```

Comment:
I don't actually think removing the comments explaining the reasoning behind the the `chmod` calls is a good thing.

Reply:
The execute bit on directories is default. It was actually only needed because of the prior `chmod -R 06xx` call, removing the execute bit from directories as well. As this is not done anymore, it could be skipped. But for now I wanted to keep the complete mode rewrite, the way it was before.

But makes sense to re-add it, as long as that `chmod` is present.